docs/custom_domains.rst

```diff
@@ -1,89 +1,112 @@
-    they do not yet allow us to acquire SSL certificates for those domains.
-    Point the CNAME to ``readthedocs.io``, with no subdomain, and re-request a certificate
-    by saving the domain in the project admin (:guilabel:`Domains` >
-    :guilabel:`Edit Domain`).
+         - If your domain has configured CAA records, please do not forget to include
+           Cloudflare CAA entries, see their `Certification Authority Authorization (CAA)
+           FAQ <https://support.cloudflare.com/hc/en-us/articles/115000310832-Certification-Authority-Authorization-CAA-FAQ>`__.
 
-    If you change the CNAME record, the SSL certificate issuance can take about
-    one hour.
+         - Due to a limitation,
+           a domain cannot be proxied on Cloudflare to another Cloudflare account that also proxies.
+           This results in a "CNAME Cross-User Banned" error.
+           In order to do SSL termination, we must proxy this connection.
+           If you don't want us to do SSL termination for your domain —
+           **which means you are responsible for the SSL certificate** —
+           then set your CNAME to ``cloudflare-to-cloudflare.readthedocs.io`` instead of ``readthedocs.io``.
+           For more details, see `this previous issue`_.
 
-.. important::
+           .. _this previous issue: https://github.com/readthedocs/readthedocs.org/issues/4395
 
-    Due to a limitation, a domain cannot be proxied on Cloudflare
-    to another Cloudflare account that also proxies.
-    This results in a "CNAME Cross-User Banned" error.
-    In order to do SSL termination, we must proxy this connection.
-    If you don't want us to do SSL termination for your domain --
-    **which means you are responsible for the SSL certificate** --
-    then set your CNAME to ``cloudflare-to-cloudflare.readthedocs.io``
-    instead of ``readthedocs.io``.
+   .. tab:: Read the Docs for Business
 
-    For more details, see this `previous issue`_.
+      In order to setup your custom domain, follow these steps:
 
-    .. _previous issue: https://github.com/readthedocs/readthedocs.org/issues/4395
+      #. Go the :guilabel:`Admin` tab of your project
+      #. Click on :guilabel:`Domains`
+      #. Enter your domain and click on :guilabel:`Add`
+      #. Follow the steps shown on the domain page.
+         This will require adding 2 DNS records, one pointing your custom domain to our servers,
+         and another allowing us to provision an SSL certificate. 
 
+      By default, we provide a validated SSL certificate for the domain.
+      The SSL certificate issuance can take a few days,
+      you can see the status of the certificate on the domain page in your project.
+
+      .. note::
+
+         Some older setups configured a CNAME record pointing to ``<organization-slug>.users.readthedocs.com``.
+         These domains will continue to resolve.
 
 Proxy SSL
 ---------
 
+.. note::
+
+   This is only available for the community version
+
```

Comment:
I think we should actually deprecate this approach. We support SSL certs natively now, and it's caused us a ton of issues in the past. I'd say lets move this to a deprecation warning.